Common/Config.cs

```diff
@@ -169,7 +169,16 @@ public IEnumerable<string> Validate()
         }
 
         private readonly List<ConfigParser.ProcessSection> Sections;
-        public bool DisableSslValidation { get; private set; } = true;
+
+        /// <summary>
+        /// Whether SSL validation should be globally disabled.
+        /// </summary>
+        public bool DisableSslValidation { get; private set; } = true;

```

Comment:
remove default value here since we always overwrite in the constructor? right now it's not clear which place specifies the default